modules/@apostrophecms/ui/ui/apos/components/AposTree.vue

```diff
@@ -72,7 +72,7 @@ export default {
   data() {
     return {
       // Copy the `rows` property to mutate with VueDraggable.
-      myRows: [],
+      myRows: this.rows,
```

Comment:
Should probably slice() that puppy, if it's in `data` you are presumably going to mutate it, and you should never mutate props.

Reply:
If you're going to mutate subproperties of it too, you should deeply clone it, but maybe that's not something you do.

Reply:
Got klona. Use klona.